apps/src/templates/sectionProgressV2/LevelProgressHeader.jsx

```diff
@@ -66,7 +66,9 @@ export default function ExpandedProgressColumnHeader({
         onClick={() => toggleExpandedChoiceLevel(level)}
       >
         {level.sublevels?.length > 0 && <FontAwesome icon="caret-right" />}
-        <div>{lesson.relative_position + '.' + level.bubbleText}</div>
+        <div>{`${lesson.relative_position}.${
+          level.isUnplugged ? 0 : level.bubbleText
```

Comment:
Since this is returned as undefined, should we instead fix this on the backend?

Reply:
I did some digging and found the `undefined` value is being set on the front end [here](https://github.com/code-dot-org/code-dot-org/blob/a78b1ef6d8cc2a88d10ccb07207791960b0236cd/apps/src/templates/progress/progressHelpers.js#L284). My only concern about changing it there would be unintended side effects on other pages.

Reply:
Seems very intentional then :\